sunpy/coordinates/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.coordinates import SkyCoord
+import sunpy.coordinates
+
```

Comment:
this needs an `__all__` list, which should only have the `great_arc` function in it.